protoduneana/verticaldrift/singlehit/runSingleHit_PDHD.fcl

```diff
@@ -52,18 +52,18 @@ physics:
 
       HitMultiplicity:      3
 
-      RadiusInt:            3    # in cm
-      RadiusExt:            6    # in cm
+      RadiusInt:            0.5   # in cm
```

Comment:
And a suggestion on fcl configurations -- since you have three run*.fcl files with similar parameter choices, you should make a shared fcl file with 

BEGIN_PROLOG 
  SingleHItCheaterDefaults: {
  }
END_PROLOG

and then #include that defaults file and have the line

ana: @local::SingleHItCheaterDefaults

Then you can override any default parameters in the individual run files.  This helps centralize maintenance in case you add a parameter someday, and it helps people run your module in their jobs and not just in your run* fcl jobs.

Reply:
See for example the Kalman filter track fitter default parameter fcl file for the pattern
https://github.com/LArSoft/larreco/blob/develop/larreco/TrackFinder/kalmanfilterfinaltrackfitter.fcl